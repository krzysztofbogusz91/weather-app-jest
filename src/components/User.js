import React, { Component } from 'react';
import PropTypes from 'prop-types';


const propTypes = {
    
};


export default class User extends Component {
    render() {
        return (
            <div className="d-flex justify-content-center align-content-center m-5">
                <h3> User Section </h3>
            </div>
        );
    }
}


User.propTypes = propTypes;



