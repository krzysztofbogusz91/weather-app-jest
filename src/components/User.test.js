import React from 'react';
import ReactDOM from 'react-dom';
import { shallow, mount } from 'enzyme';
import set from '../enzyme_set_up';
import User from './User';

describe('User',()=>{
    const us = shallow( <User /> )

    it('renders without crashing', () => {
        expect(us).toMatchSnapshot();
    });
      
})
