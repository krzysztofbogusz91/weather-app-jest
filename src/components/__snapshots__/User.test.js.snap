// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <User />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h3>
         User Section 
</h3>,
      "className": "d-flex justify-content-center align-content-center m-5",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": " User Section ",
      },
      "ref": null,
      "rendered": " User Section ",
      "type": "h3",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h3>
           User Section 
</h3>,
        "className": "d-flex justify-content-center align-content-center m-5",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": " User Section ",
        },
        "ref": null,
        "rendered": " User Section ",
        "type": "h3",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
